docs/CONTRIBUTING.md

```diff
@@ -34,10 +34,46 @@ Some code guidelines to keep in mind when contributing to laminar or amethyst-ne
     - Keep comments small
     - Don’t create unnecessary comments. They must add value
     - Comments should explain the “why” not the “what”
+    - All `///` comments should start capitilized and end with an 'dot'.
+    - Function comments should be plural like: 'Returns', 'Creates', 'Instantiates' etc.
+    - All `//` explain code 'in' functions should have no capital letter and not 'dot' 
+    - Referenced types, functions, variables should be put inside '`code markup`'. 
 2. Hard Coding
     - Don't hard code values anywhere
     - Use the ‘NetworkConfig’ type for common network settings, use consts or parameter input
     - Use of lazy_static is acceptable but first make sure you can’t fix the issue in other ways
 3. Code markup
     - Keep files small. Better have small files with small pieces of logic than having one file with 1000 lines of logic with multiple types/structs etc. Note that I speak of logic, tests not included
 	- No panics/unwraps in the main codebase, but they are accepted in tests
+
+## Import Reordering
+All imports are semantically grouped and ordered. The order is:
```

Comment:
Do we have a way to enforce this? It seems awfully pedantic if we don't have a programmatic way of both performing the transform and validating it.

Reply:
I don't believe so. However, as noted the IntelIJ rust plugin is able to do this. So once in a while, we can run that to validate everything is ordered correctly. I have no idea how to enforce it in some way.